src/styles/code.css

```diff
@@ -0,0 +1,8 @@
+.information__code {
```

Comment:
same here, it's an element of information block, no need for external file, leave files only for blocks and its elements